src/index.html

```diff
@@ -30,7 +30,7 @@
       }
     </style>
   </head>
-  <body>
+  <body style="overflow: hidden;">
```

Comment:
So scroll bars don't show up if the content ends up overflowing the app window.
- This might need to change down the road for accessibility reasons...